l2_vars_and_types/homework/year.py

```diff
@@ -0,0 +1,2 @@
+age = int(input('Enter your age:\n' '\n'))
+print('\n'f'You were born in {2018 - age} year!')
```

Comment:
use `f` before the whole string
`print(f'\nYou were born in {2018 - age} year!')`